pom.xml

```diff
@@ -831,6 +831,22 @@
                 <version>1.9.1</version>
             </dependency>
 
+            <dependency>
+                <groupId>io.confluent</groupId>
```

Comment:
What is the licence of this?

Reply:
Apache License 2.0:
https://github.com/confluentinc/schema-registry/blob/master/client/src/main/java/io/confluent/kafka/schemaregistry/client/SchemaRegistryClient.java